docs/_templates/2022/index.html

```diff
@@ -0,0 +1,207 @@
+    <div class="uk-container">
+      <p style="vertical-align: middle;">Want to receive conference and community updates?</p>
+      <a class="uk-button uk-button-announcement" href="{{ pathto('conf/' + shortcode + '/' + year_str + '/mailing-list' ) }}">
+        Join our Mailing List
+      </a>
+    </div>
+  </div><!--- end announcement -->
+
+  {% endif %}
+
+  <div class="about">
+    <div class="uk-container">
+
+      <div class="about-content">
+        <h2>What is Write the Docs?</h2>
+        <p>
+          Write the Docs brings everyone who writes the docs together in the same room: Programmers, Tech Writers, Support, Designers, Developer Advocates, and more. We all have things to learn from each other, and there’s no better way than sitting together and talking.
+        </p>
+        <p>
+          We invite you to join {{ about.attendees }} other folks in our event to explore the art and science of documentation.
+        </p>
+        {% if flaglivestreaming %}
+        <p>
+          <a href="{{ pathto('conf/' + shortcode + '/' + year_str + '/livestream') }}">Live streaming is available</a> during the conference.
+          You do <strong>not</strong> need a ticket to watch the stream. Videos of talks will be published after the conference.
+        </p>
+        {% endif %}
+
+        {% if cobrand is defined %}
+        <h2>What is Open Help?</h2>
+        <p>
+          The Open Help Conference features two days of presentations and open discussions. In true open source spirit, our discussions and agenda are led by our attendees, allowing you to learn and share what matters to you most. With some of the most interesting people in the industry, the Open Help Conference showcases the ideas that are shaping the future of help.
+      </p>
+
+        <h2>What is this event?</h2>
+        <p>
+          This event combines the two communities and conferences. We will have one day of Write the Docs-style talks, followed by one day of combined Write the Docs and Open Help discussions. Then we'll have an Open Help-style three-day documentation sprint.
+        </p>
+        {% endif %}
+
+      </div>
+
+      <div class="picture-set">
+        <div data-aos="pictrue-03" data-aos-once="true">
+        <img src="{{ pathto('_static/conf/images/pics/pictrue-03.jpg', 1) }}">
+        </div>
+        <div data-aos="pictrue-02" data-aos-once="true">
+        <img src="{{ pathto('_static/conf/images/pics/pictrue-02.jpg', 1) }}">
+        </div>
+        <div data-aos="pictrue-01" data-aos-once="true">
+        <img src="{{ pathto('_static/conf/images/pics/pictrue-01.jpg', 1) }}">
+        </div>
+      </div>
+    </div>
+  </div><!-- end about -->
+
+  <div class="schedule-overview">
+    <div class="uk-container" uk-grid>
+      <h2>Schedule Overview</h2>
+      <div class="schedule-item uk-width-1-3@s">
+        <img style="height: 150px;" src="{{ pathto('_static/conf/images/icons/' + date.day_one.icon + '-' + color +  '.svg', 1) }}">
```

Comment:
Should we be explicit about writing day / no writing day / hybrid writing day? here?

Reply:
Brainstorm thought: keep Writing Day, but make it remote-only?

aka, for now, call "Writing Day" TBD?

Reply:
Yea, I'm a little skeptical about how much we're thinking about changing things up, given that we know our existing formula has worked OK for online & in-person events. We don't need to decide on that now though.